client/main/vareditor/variablemodel.js

```diff
@@ -54,7 +66,9 @@ const VariableModel = Backbone.Model.extend({
         filterNo: -1,
         importName: '',
         trimLevels: true,
-        autoApply: true
+        autoApply: true,
+        transform: 0, // zero means 'none'
+        parentId: -1
```

Comment:
use `0` for none for `parentId` too. -1 takes up four bytes over the wire, 0 takes up none.